### PR TITLE
[nl]: Dutch translations

### PR DESCRIPTION
**Pull request by chillbram**
Unsure about the pluralization of some of the actions, but at least this is much improved from the machine translated ones

### CODE REVIEW

File: locales/nl/_excludes.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[
+    "Details",
+    "details",
```
Comment (andrey-helldar) [inline locales/nl/_excludes.json:3]: A key by that name does not exist.

File: locales/nl/_excludes.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    "Open",
+    "open"
```
Comment (andrey-helldar) [inline locales/nl/_excludes.json:5]: A key by that name does not exist.

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: locales/nl/_excludes.json (hunk 1 above)
Comment (chillbram) [reply locales/nl/_excludes.json:3]: I added them based on https://laravel-lang.com/statuses-actions-nl.html#statuses-actions-nl-php where there is a `Details` in the json and a `details` in the php. Should the PHP ones be removed?

Comment (andrey-helldar) [reply locales/nl/_excludes.json:3]: It seems to have found another problem - keys are only checked against the JSON file avoiding checking against the PHP file.

I will fix this.

**Review by andrey-helldar — APPROVED** [review]
Dankjewel

**Comment by andrey-helldar** [comment]
I completely forgot that the exception file contains values, not keys 😅

For example:

```json
// json
{
  "foo": "qwerty"
}

// php
{
  "bar": "qwerty"
}

// _excludes.json
{
  "qwerty"
}
```

**Pull request merged**